packages/camera/camera_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: camera_platform_interface
+description: A common platform interface for the camera plugin.
+homepage: https://github.com/flutter/plugins/tree/master/packages/camera/camera_platform_interface
+# NOTE: We strongly prefer non-breaking changes, even at the expense of a
+# less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
+version: 1.0.0
+
+dependencies:
+  flutter:
+    sdk: flutter
+  meta: ^1.0.5
+  plugin_platform_interface: ^1.0.1
+  file_selector_platform_interface: ^1.0.0
```

Comment:
We need to publish cross_file before this PR, so this dependency can be updated.

Reply:
We replaced `file_selector_platform_interface` with the published version of `cross_file`.